LayoutKitTests/TextViewLayoutTests.swift

```diff
@@ -0,0 +1,169 @@
+
+            XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize, "fontName:\(textTestCase.font?.fontName) text:\(textTestCase.text) fontSize:\(textTestCase.font?.pointSize)")
+        }
+    }
+
+    func testMutipleLinesWithUnattributedString() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+
+        let textView = UITextView(text: text, font: UIFont.systemFont(ofSize: 12))
+
+        let layout = TextViewLayout(text: text, font: UIFont.systemFont(ofSize: 12))
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testMutipleLinesWithAttributedString() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let attributedString = NSAttributedString(
+            string: textString)
+        let attributedText = Text.attributed(attributedString)
+
+        let textView = UITextView(text: attributedText)
+
+        let layout = TextViewLayout(text: attributedText)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testMutipleLinesWithMutipleAttributedStrings() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let attributedString1 = NSMutableAttributedString(
+            string: textString,
+            attributes: [NSFontAttributeName: UIFont.helvetica(size: 15)])
+        let attributedString2 = NSMutableAttributedString(
+            string: textString,
+            attributes: [NSFontAttributeName: UIFont.helvetica(size: 12)])
+        attributedString1.append(attributedString2)
+        let attributedText = Text.attributed(attributedString1)
+
+        let textView = UITextView(text: attributedText)
+
+        let layout = TextViewLayout(text: attributedText)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testLineFragmentPadding() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
+        let text = Text.unattributed(textString)
+        let lineFragmentPadding: CGFloat = 30.0
+
+        let textView = UITextView(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            lineFragmentPadding: lineFragmentPadding)
+
+        let layout = TextViewLayout(
+            text: text,
+            font: UIFont.systemFont(ofSize: 12),
+            lineFragmentPadding: lineFragmentPadding)
+        let layoutView = layout.arrangement()
+
+        XCTAssertEqual(layoutView.frame.size, textView.intrinsicContentSize)
+    }
+
+    func testTextContainerInset() {
+        let textString = "Hello World\nHello World\nHello World\nHello World\nHello World"
```

Comment:
It would be good to add a test that combines word wrapping with textContainerInset to confirm that text gets wrapped in a consistent way.